test/models/erratum_test.rb

```diff
@@ -126,5 +127,12 @@ def test_installable_for_systems
       refute_includes errata, @bugfix
       refute_includes errata, @enhancement
     end
+
+    def test_installable_for_systems_dev_environment
+      errata = Erratum.installable_for_systems([@errata_server_dev, @simple_server])
+      assert_includes errata, @security
+      refute_includes errata, @bugfix
+      refute_includes errata, @enhancement
+    end
```

Comment:
this test case doesn't actually test for the sql issue.  I think you'd need to an in_repositories query onto it as well.